index.js

```diff
@@ -25344,14 +25344,14 @@ function BeeSwarmSimulator(DATA){
             }
              
              
-            if(e.key==='i') inventoryButton.onclick()
-            if(e.key==='q') questButton.onclick()
-            if(e.key==='b') beesButton.onclick()
+            if(e.key==='i' || e.key==='I') inventoryButton.onclick()

```

Comment:
In this logic, I cause the activation of the trigger whether the lowercase or uppercase version of the key is pressed. This ensures that the input is case insensitive and that having caps lock on will not affect game play.